frontend/build.gradle.kts

```diff
@@ -15,33 +15,49 @@ tasks {
         enabled = false
     }
 
-    val npm_run_build by getting {
-        dependsOn("yarn_install")
+    val yarn_install by getting {
+        inputs.file("package.json")
+        inputs.file("yarn.lock")
+        outputs.dir("node_modules")
+    }
+
+    val deleteMomentLocales by creating(Delete::class) {
+        dependsOn(yarn_install)
+        delete(fileTree("${projectDir}/node_modules/moment/locale") {
+            include("*.js")
+        })
```

Comment:
Ha! Never thought about doing that, cool.

Reply:
It's a bit of a hack, but it's a simple and pragmatic way of getting rid of the useless locales. I would have preferred to configure webpack to ignore these locales, but ng-cli stays desperately unconfigurable.